src/app/core/standard-guidelines-module/standard-guidelines.service.ts

```diff
@@ -47,7 +46,7 @@ export class GuidelineService {
       ).toPromise();
   }
 
-  private formatFilter(filter) {
+  private formatFilter(filter: any) {
```

Comment:
Same question here.

You can refer to standard-guidelines to check out what it is expecting.